pep_sphinx_extensions/pep_theme/static/style.css

```
@@ -281,6 +281,13 @@ table.pep-zero-table tr td:nth-child(5) {
     width: 50%;
 }
 
+/* Numerical index */
+article:has(> section#numerical-index) {
+    float: unset !important;
+    width: 100% !important;
+    max-width: 100% !important;
```

It feels a bit too wide currently, how about?:
```suggestion
    width: 90% !important;
    max-width: 90% !important;
```